src/rstar.jl

```diff
@@ -21,17 +23,16 @@ verbosity level.
 using MLJModels
 XGBoost = @load XGBoostClassifier
 
-# Compute 20 samples of the R* statistic using sampling from according to the prediction probabilities.
+# Compute 20 samples of the R* statistic using sampling according to the prediction probabilities.
 Rs = rstar(XGBoost(), chn; iterations=20)
 
 # estimate Rstar
 R = mean(Rs)
 
 # visualize distribution
+using StatsPlots
 histogram(Rs)
```

Comment:
I don't think it is helpful to use `StatsPlots` or `histogram` here in the documentation. In principle, on Julia >= 1.3, this could be made a doctest otherwise.